oauth2/fosite_store_memory.go

```diff
@@ -182,3 +184,17 @@ func (s *FositeMemoryStore) RevokeAccessToken(ctx context.Context, id string) er
 	}
 	return nil
 }
+
+func (s *FositeMemoryStore) CleanseTokens(ctx context.Context, before time.Time) error {
+	errs := multierror.Accumulator{}
+
+	for sig, token := range s.AccessTokens {
+		if token.GetRequestedAt().Before(before) {
+			errs.Push(s.DeleteAccessTokenSession(ctx, sig))
```

Comment:
Return all the errors. The accumulator returns an error containing a list of errors